exotic/exotic.py

```diff
@@ -1904,24 +1904,25 @@ def main():
             pass
 
         if fileorcommandline == 1:
-            userpDict['pName'] = str(input("\nEnter the Planet Name: "))
+            userpDict['pName'] = str(input("\nEnter the Planet Name. Make sure it matches the case sensitive name used on Exoplanet Archive (https://exoplanetarchive.ipac.caltech.edu/index.html): "))
 
         nea_obj = NASAExoplanetArchive(planet=userpDict['pName'])
         userpDict['pName'], CandidatePlanetBool, pDict = nea_obj.planet_info()
 
         # observation date
         if fileorcommandline == 1:
-            infoDict['date'] = str(input("\nEnter the Observation Date: "))
+            infoDict['date'] = str(input("\nEnter the Observation Date (XX-XX-XXXX): "))
```

Comment:
I would actually change this from "XX-XX-XXXX" to "MM-DD-YYYY" for our international collaborators.